CCL.Creator/Validators/CarValidators.cs

```diff
@@ -202,23 +202,23 @@ private static IEnumerable<Result> CheckLiveryColliders(CustomCarVariant livery)
 
             // bounding collider
             var collision = collidersRoot.FindSafe(CarPartNames.COLLISION_ROOT);
-            var collisionComp = collision ? collision!.GetComponent<BoxCollider>() : null;
-            if (!(collision && collisionComp))
+            var collisionComp = collision ? collision!.GetComponentsInChildren<BoxCollider>(true) : Enumerable.Empty<Collider>();
```

Comment:
`Collider` instead of `BoxCollider`, or keep it like this?

Reply:
I think that [collision] needs to always be a box collider